dali/dfu/dfuwu.cpp

```diff
@@ -51,9 +51,20 @@ static StringBuffer &getXPathBase(StringBuffer &wuRoot)
     return wuRoot.append("/DFU/WorkUnits");
 }
 
+static StringBuffer &getXPath(StringBuffer &wuRoot, const char *wuid, unsigned int task)
+{
+    getXPathBase(wuRoot);
+    if (task)
+        return wuRoot.append('/').append(wuid).append("/Tasks/").append(wuid).append("T").append(task);
+    return wuRoot.append('/').append(wuid);
+}
+
 static StringBuffer &getXPath(StringBuffer &wuRoot, const char *wuid)
 {
     getXPathBase(wuRoot);
+    const char *task = strchr(wuid, 'T');
+    if (task)
+        return wuRoot.append('/').append(task-wuid, wuid).appendf("/Tasks/").append(wuid);
```

Comment:
Because the task is actually a fully functioning DFU COPY workunit the child tag has a name with both parent wuid + task id.  It ends up looking like:

```
<P20220425-173239>
  <Tasks>
     <P20220425-173239T1>...</P20220425-173239T1>
     <P20220425-173239T2>...</P20220425-173239T2>
 </Tasks>
</P20220425-173239>

```
It allows for easy discovery of parent and child, and also allows existing code to continue to work.